main.go

```diff
@@ -99,10 +100,16 @@ func main() {
 		log.Fatal(err)
 	}
 
+	policy, exists := plan.Policies[cfg.Policy]
```

Comment:
We have that at multiple places. Let's make a general sweep through the code for `v0.4`.